CONTRIBUTING.md

```diff
@@ -41,60 +47,57 @@ potentially received objections/suggestions to be handled.
-made to the design and proposed specification edits to accurately represent
-the current status of the proposal.
-
-If the proposal adds or changes an object model or resource endpoint definition,
-it is expected that the PR will also include the necessary updates to the
-[Open API document](openapi.yaml).
+be done by pointing to a branch in a Github repo with the proposed edits or by
+creating a PR with the `do not merge` label assigned to it. Reviewers, or
+potential implementers of the feature, can then easily see the exact changes
+being proposed so that they can comment on them. If the proposal adds or changes
+an object model or resource endpoint definition, it is expected that the PR will
+also include the necessary updates to the [Open API document](openapi.yaml) and
+[Swagger document](swagger.yaml).
 
 Once support for the change has been implemented, a PR should be sent to this
-repo for the change to the Open Service Broker API specification. By this
-point, the API interactions should be well understood and there should be no
-technical surprises; we expect the only discussion necessary on PRs to be for
-wordsmithing and formatting.
+repo for the change to the Open Service Broker API specification.
 
-## GitHub Labels
+### Proposal Stages
 
-When new [design proposals](#design-change-proposals) for the API specification
-are submitted, the working group will use a set of pre-defined GitHub labels to
-highlight the current stage of the proposal. Note that these labels will not be
+Major changes will be tracked through a number of stages before becoming part of
+the specification. The working group will use a set of pre-defined Github labels
+to highlight the current stage of each proposal. Note that these labels will not be
 used for [minor changes](#minor-changes).
 
-The labels that will be used are as follows:
-- `1 - reviewing proposal`: The API working group is actively reviewing a
-proposal that has been submitted as a GitHub Issue with the aim of validating
-both the problem statements and any proposed solutions.
-- `2 - validating through implementation`: One of more platforms are actively
-working on the proposal with the aim of providing feedback on the implementation
-of the proposed solution.
-- `3 - reviewing PR`: Feedback has been received on the implementation of the
-proposed solution and a Pull Request has been created containing the validated
-specification changes.
-
-Note that not all issues will require a validation through implementation stage,
-and proposals can move back to a previous label at any time. When a PR is
-merged, the associated issue should be closed and any labels removed from the
-issue.
+The stages that a proposal will go through are as follows:
+- `1 - reviewing proposal`:
+  The API working group is actively reviewing a proposal that has been submitted
+  as a Github issue with the aim of validating both the problem statements and
```

Comment:
isn't this process normally for PRs not issues?

Reply:
We say that design changes have to have a google doc (via an issue)